plugins/org/src/components/Cards/OwnershipCard/OwnershipCard.test.tsx

```diff
@@ -132,6 +133,17 @@ describe('OwnershipCard', () => {
       },
     );
 
+    expect(catalogApi.getEntities).toHaveBeenCalledWith({
```

Comment:
Original unit tests returned all entities no matter what the filter was. I left it this was. But I want to ensure the "backwards" Component&API kinds in filter. Is this the correct way? Or shall I implement the filter mock, e.g.

```typescript
catalogApi.getEntities.mockImplementation(
      (
        request?: CatalogEntitiesRequest,
        options?: CatalogRequestOptions,
      ): Promise<CatalogListResponse<Entity>> => {
        if (options === null) {
          throw 'options';
        }
        if (request === null) {
          throw 'request';
        }
        return Promise.resolve({
          items: items.filter(item => #TODO: filter item.kind based on request.kind ???,
            ),
          ),
        } as CatalogListResponse<Entity>);
      },
    );
```

Reply:
@benjdlambert WDYT?

Reply:
Yeah let's add a test like that I think just to cover our bases. 👍

Reply:
Ok, done.